### ISSUE TITLE
golden items only give 1 nugget, some iron items give only 1 nugget

### ISSUE THREAD

**Issue by joseph-peace** [LepkoQQ/Recycler#1]
pretty much what the title says
diamond items work fine though

tested on minecraft 1.14.4
forge 1.14.4 - 28.1.0
mod version 1.0.0

this is a cool mod
keep up the good work

**Comment by LepkoQQ** [comment]
I noticed this early in development and I thought i fixed it. Were there any other mods installed? Maybe you could post a log so I can see what happens at startup.

**Comment by erney519** [comment]
hi  is this log you say ?
[26九月2019 02:00:58.346] [Client thread/DEBUG] [cpw.mods.inventorysorter.InventorySorter/]: Skipping action handling for blacklisted slot
[26九月2019 02:01:05.743] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Erney519: 已將 1 個 [黃金胸甲] 給予 Erney519]
[26九月2019 02:01:05.760] [Client thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] 已將 1 個 [黃金胸甲] 給予 Erney519
[26九月2019 02:01:11.290] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Erney519: 已將 1 個 [木鎬] 給予 Erney519]
[26九月2019 02:01:11.298] [Client thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] 已將 1 個 [木鎬] 給予 Erney519
[26九月2019 02:01:12.358] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: ########## GL ERROR ##########
[26九月2019 02:01:12.358] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: @ Render
[26九月2019 02:01:12.358] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: 65539: Invalid key 0
[26九月2019 02:01:22.559] [Client thread/DEBUG] [cpw.mods.inventorysorter.InventorySorter/]: Skipping action handling for blacklisted slot
[26九月2019 02:01:22.559] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: ########## GL ERROR ##########
[26九月2019 02:01:22.559] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: @ Render
[26九月2019 02:01:22.559] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: 65539: Invalid key 0
[26九月2019 02:01:26.112] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: ########## GL ERROR ##########
[26九月2019 02:01:26.112] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: @ Render
[26九月2019 02:01:26.112] [Client thread/ERROR] [net.minecraft.client.MainWindow/]: 65539: Invalid key 0
[26九月2019 02:01:26.765] [Server thread/INFO] [net.minecraft.server.integrated.IntegratedServer/]: Saving and pausing game...

**Comment by coolsimulations** [comment]
Using 1.0.0 and Forge 1.14.4-28.1.20.
In detail the items results:

Iron Shovel (Unused): _1 Iron Ingot_
Iron Pickaxe (Unused): _1 Iron Nugget_
Iron Axe (Unused): _1 Iron Nugget_
Iron Sword (Unused): _2 Iron Ingots_
Iron Hoe (Unused): _1 Iron Nugget_
Iron Helmet (Unused): _1 Iron Nugget_
Iron Chestplate (Unused): _1 Iron Nugget_
Iron Leggings (Unused): _1 Iron Nugget_
Iron Boots (Unused): _1 Iron Nugget_
Minecart: _Doesn't Smelt Anything_
Iron Horse Amour: _6 Iron Ingots_

Gold Shovel (Unused): _1 Gold Nugget_
Gold Pickaxe (Unused): _1 Gold Nugget_
Gold Axe (Unused): _1 Gold Nugget_
Gold Sword (Unused): _1 Gold Nugget_
Gold Hoe (Unused): _1 Gold Nugget_
Gold Helmet (Unused): _1 Gold Nugget_
Gold Chestplate (Unused): _1 Gold Nugget_
Gold Leggings (Unused): _1 Gold Nugget_
Gold Boots (Unused): _1 Gold Nugget_
Gold Horse Amour: _1 Gold Nugget_

All Diamonds work perfectly though.

**Comment by l0lm3n** [comment]
still having this issue! anyone got a update yet?

**Comment by LepkoQQ** [comment]
yeah, there seems to be and issue with vanilla recipes taking priority. Don't have time to look in to it right now, hopefully soon, sorry guys.
